src/AerospikeCache.php

```diff
@@ -104,14 +100,23 @@ protected function doDelete(array $ids): bool
         return $removedAllItems;
     }
 
+    /**
+     * @param int $lifetime
+     * @return string[] keys of values that failed during save operation
+     */
     protected function doSave(array $values, $lifetime): array
     {
         $failed = [];
         foreach ($values as $key => $value) {
             $data = [self::WRAPPER_NAME => $value];
-            $statusCode = $this->aerospike->put($this->createKey($key), $data, $lifetime, [\Aerospike::OPT_POLICY_KEY => \Aerospike::POLICY_KEY_SEND]);
+            $statusCode = $this->aerospike->put(
+                $this->createKey($key),
+                $data,
+                $lifetime,
+                [\Aerospike::OPT_POLICY_KEY => \Aerospike::POLICY_KEY_SEND]
+            );
             if ($statusCode !== \Aerospike::OK) {
-                $failed[] = $key;
+                $failed[] = (string) $key;
```

Comment:
Isn't the `$key` always `string`? Why do you cast it to string?

Reply:
Because of PHP stan.

Reply:
> Because of PHP stan.

PHP stan just tell, there is type error .. 

I'd like to know, **why** it complains.. 
I've checked it by myself and $key must be always string, so the casting is redundant and simple annotation (`* @param array<string, mixed> $values`) will satisfy a phpstan as well. And client of this api will know, what should be passed in. If the key is anything but `string`, it will crash in `createKey` method, so it should be annotated at least.

Reply:
removed the cast to string. Added the comment